runner/runner/main.py

```diff
@@ -212,13 +219,12 @@ def load_monado(config: Mapping[str, Any]) -> None:
     enable_alignment_flag = config["enable_alignment"]
     realsense_cam_string = config["realsense_cam"]
 
-    is_mainline: bool = bool(config["action"]["is_mainline"])
-    build_runtime(config, "so", is_mainline=is_mainline)
+    is_vulkan: bool = bool(config["action"]["is_vulkan"])
+    build_runtime(config, "so", is_vulkan = True)
 
     def process_plugin(plugin_config: Mapping[str, Any]) -> Path:
-        if is_mainline:
-            plugin_config.update(ILLIXR_MONADO_MAINLINE="ON")
-        return build_one_plugin(config, plugin_config)
+        plugin_config.update(ILLIXR_MONADO="ON")
```

Comment:
Is this redundant in the build_one_plugin?

Reply:
I haven't changed this part since I copied Jeffrey's script.

Reply:
This is fine, some plugins (e.g. timewarp_gl) might want to use ifdefs on their own. Although we may potentially want to find a better solution in the future.